.golangci.yml

```diff
@@ -34,7 +34,7 @@ linters-settings:
   goimports:
     local-prefixes: github.com/apache/yunikorn
   govet:
-    check-shadowing: true
+    shadow: true
```

Comment:
This is the new format for shadow?

Reply:
Yes, got tired of the warning and didn't figure it was worthy of its own PR.